README.md

```diff
@@ -1,2 +1,61 @@
-# Hello-world
+# Project Title
+Hello-world
+
+![image](https://github.com/the-faizmohammad/Hello-world/assets/134736318/cc2bf848-5178-4db0-ab9b-641bc245c2f7)
+
+## Project Description
 Getting Familiar with Github , repo, branches &amp; code review.
+
+## Table of Contents
+
+- [Installation](#installation)
+- [Usage](#usage)
+- [Contributing](#contributing)
+- [Credits](#credits)
+- [License](#license)
+- [Contact](#contact)
+- [Acknowledgements](#acknowledgements)
+
+## Installation
+
+1. Clone the repository:
+
+2. Open the project directory:
+
```

Comment:
- Kindly use [Microverse Readme](https://github.com/microverseinc/readme-template) provided here and please make it descriptive and personalize it where requires. 
-  Please add a descriptive summary to your Pull request. This makes your PR professional and it also helps anyone visiting your GitHub to understand what you have done in your PR